### DISCUSSION TITLE
League Flysystem 2 support

### DISCUSSION THREAD

**Discussion by joshbruce** [laravel/framework#38548] [Ideas] [answered]
Just ran into a dependency issue. Another library I use makes use of https://github.com/thephpleague/flysystem 2+

Laravel is only compatible with https://github.com/thephpleague/flysystem 1+

Not sure about the internals around Laravel, so, not sure if this is on the radar or roadmap; either multi-support or switching to 2+.

**Comment by woodspire** [comment, marked as answer]
https://github.com/laravel/framework/pull/33612

It seems it is done only on the master branch that will be the base of Laravel 9. It seems Laravel 8 will only support version 1.

**Reply by joshbruce** [reply]
Thanks! Appreciate you making the time to look, respond, and telling me where you looked.
